### ISSUE TITLE
main branch : not showing the quotes

### ISSUE THREAD

**Issue by arademaker** [own-pt/sensetion.el#125]
simple and double quotes marking examples are not shown in the interface.

**Comment by odanoburu** [comment]
tbh this doesn't bother me since we wanted to get rid of the quotes anyway, plus that's not a problem in `dev`. but I've opened a PR if you'd like a fix.

**Comment by arademaker** [comment]
I think it is important, for instance, imagine a quote suggesting a non-conventional interpretation of a word.  But as always, only after seeing the results I will be able to judge the readability.  I suggest adding it as an option show-quotes on/ff.

**Comment by odanoburu** [comment]
you're right, the display of this synset is confusing: http://wnpt.sl.res.ibm.com/wn/synset?id=01156750-a
